app/app.go

```diff
@@ -140,3 +149,63 @@ func addData(management *config.ManagementContext, cfg Config) error {
 
 	return addMachineDrivers(management)
 }
+
+func getZombiePid(path string) int {
+	// Ignore errors
+	content, err := ioutil.ReadFile(path)
+	if err != nil {
+		return 0
+	}
+
+	fields := strings.Split(string(content), ") ")
+	fields = strings.Split(fields[len(fields)-1], " ")
+
+	if fields[0] == "Z" && fields[1] == "1" {
+		i, _ := strconv.Atoi(strings.Split(string(content), " ")[0])
+		return i
+	}
+
+	return 0
+}
+
+func reaper() {
+	reap := make([]string, 0, 5)
+
+	for {
+		time.Sleep(1 * time.Second)
```

Comment:
Why do we need to do this every second?

Reply:
:) because i copied the whole method darren gave me

Reply:
To be discussed...